src/Infrastructure.Dapper/Repositories/UserRepository.cs

```diff
@@ -115,8 +129,15 @@ public async Task<DateTime> GetAccountRevisionDateAsync(Guid id)
         }
     }
 
+    public override async Task<User> CreateAsync(User user)
+    {
+        ProtectData(user);
+        return await base.CreateAsync(user);
+    }
+
     public override async Task ReplaceAsync(User user)
     {
+        ProtectData(user);
```

Comment:
Do we ever use the `user` after calling this method? This will modify the original user and protect the fields. Should we clone the user and protect the clone to ensure the returned value or original user isn't protected?

Reply:
That's a good point. Not sure what side effects that might cause, but if the application saved the same user object twice it would cause problems.

Reply:
I think my latest commit should address this.